docs/design/2021-04-26-lock-view.md

```diff
@@ -76,6 +78,21 @@ Several tables will be provided in `information_schema`. Some tables has both lo
 * Collecting, storing and querying:
     * All these will be collected on TiKV LockManager, and will need a new RPC entry for TiDB to query. LockManager won't store the un-hashed key or SQL_DIGEST for now, so we need to modify it.
 * The SQL Digest of the transaction that's currently holding the lock may be helpful, but it's hard to implement under the current architecture. So it won't be included in the first version of the feature.
+* `KEY_INFO` is the decoded information about the key, like:
```

Comment:
I'm not absolutly sure about whether we should represent it as a json. Is it possible that the user may want to do joins on `table_id` or some other fields.

Reply:
It's easy to get the table_id from the json by using `JSON_EXTRACT` function. I think it should be fine. If you expand it to separated columns for these fields, it will need too many columns to show the full information, and usually some of the columns are not used (for example index_id, index_name, index_value, ... are not used for row keys). IMHO I think json is just fine, and you can omit the not-used fields from the json.

By the way, the following example is just my draft. If necessary you can adjust the structure.